### ISSUE TITLE
Executorch OSS CI tracking issue (2023)

### ISSUE THREAD

**Issue by huydhn** [pytorch/executorch#8]
I'm creating this initial issue to keep track of executorch OSS CI progress in 2023.  P0 needs to be done before Oct.

* [x] (**P0**) Integrate executorch with GHA and PyTorch-org shareable runners (AWS, linux).  This is needed to provide the infrastructure to run OSS CI.
* [x] (**P0**) Integrate executorch with GitHub webhook and Rockset DB.  This is needed to keep track of which is run on OSS CI.
* [x] (**P0**) Setup e2e build and test on Linux runner.  This requires BUCK OSS to run.
    * [x] (**P0**) Check with Executorch team if they need support on getting the BUCK OSS setup.
    * [x] (**P0**) Also setup one for Mac M1
    * [x] (**P0**) Add e2e testing for the rest of the sample models T160756042
* [ ] (P1) Setup devices (Android, iOS) to run benchmark tests.  The prefer option is https://aws.amazon.com/device-farm
* [x] (P1) Sync with Executorch and TorchBench on how to benchmark Executor.
* [ ] (P1) Setup benchmark CI for Executor.
* [ ] (P1) Add OSS alert for Executorch when CI workflow fails

----
This is probably not needed for now, only list them here for reference.

* [x] (P2) Linter
* [ ] (P2) Binaries release
* [ ] (P2) GitHub First features such as merge bot, Dr.CI

cc @malfet @seemethere @mergennachin @dbort @clee2000

**Comment by huydhn** [comment]
Executorch goes public today, so we can close this one as the first phase is done
